lib/workers/repository/error.ts

```diff
@@ -91,6 +91,7 @@ export default async function handleError(
   }
   if (err.message === REPOSITORY_FORKED) {
     logger.info('Repository is a fork and not manually configured - skipping');
+    logger.info('See: https://docs.renovatebot.com/configuration-options/#includeforks');
```

Comment:
remove and just add the config option to check to the log above, as the url can change.

Reply:
Done.